component/site/views/default/list/tmpl/csvprintevents.php

```diff
@@ -24,10 +26,15 @@
 
 $fields = array();
 
+$app = JFactory::getApplication();
+$currentMenuId = JSite::getMenu()->getActive()->id;
+$mparams   = $app->getMenu()->getItem($currentMenuId)->params;
+
```

Comment:
We cheat in JEvents so that JComponentHelper::getParams("com_jevents"); picks up the menu item params by default - see the frontend jevents.php line 61-90